### PR TITLE
docs: add documentation for ADMIN ALTER DDL JOBS command

### PR DESCRIPTION
**Pull request by tangenta**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/19085
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by fzzf678 — APPROVED** [review]

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 1]
```diff
@@ -126,31 +133,32 @@ ADMIN SHOW DDL JOBS;
...
+14 rows in set (0.00 sec)
 ```
 
 From the output above:
 
-- Job 59 is currently in progress (`STATE` of `running`). The schema state is currently in `write reorganization`, but will switch to `public` once the task is completed to note that the change can be observed publicly by user sessions. The `end_time` column is also `NULL` indicating that the completion time for the job is currently not known.
+- Job 565 is currently in progress (`STATE` of `running`). The schema state is currently in `write reorganization`, but will switch to `public` once the task is completed to note that the change can be observed publicly by user sessions. The `end_time` column is also `NULL` indicating that the completion time for the job is currently not known.
 
-- Job 60 is an `add index` job, which is currently queued waiting for job 59 to complete. When job 59 completes, the `STATE` of job 60 will switch to `running`.
+- Job 566 is an `add index` job, which is currently queued waiting for job 565 to complete. When job 565 completes, the `STATE` of job 60 will switch to `running`.
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:161]:
```suggestion
- The `STATE` for `job_id` 566 is shown as `queueing`, indicating that it is queuing. When job 565 completes and job 566 begins execution, the `STATE` for job 566 will change to `running`.
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 2]
```diff
@@ -91,6 +89,15 @@ The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the curre
...
+    - `txn`: basic transactional backfill.
+    - `txn-merge`: transactional backfilling with a temporary index that gets merged with the original index when the backfilling is finished.
+    - `DXF`: tasks executed with distributed frameworks configured via [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
+    - `service_scope`: the service scope of the TiDB node configured via [`tidb_service_scope`](/system-variables.md#tidb_service_scope-new-in-v740).
+    - `thread`: concurrency of backfill tasks. The initial value can be set with `tidb_ddl_reorg_worker_cnt`. It supports dynamic modification via `ADMIN ALTER DDL JOBS`.
+    - `batch_size`: batch size of backfill tasks. The initial value can be set with `tidb_ddl_reorg_batch_size`. It supports dynamic modification via `ADMIN ALTER DDL JOBS`.
+    - `max_write_speed`: flow control during ingest task import. The initial value can be set with `tidb_ddl_reorg_max_write_speed`. It supports dynamic modification via `ADMIN ALTER DDL JOBS`.
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:100]:
```suggestion
    - `max_write_speed`: flow control during ingest task import. The initial value can be set with `tidb_ddl_reorg_max_write_speed`. It supports dynamic modification via [`ADMIN ALTER DDL JOBS`](sql-statements/sql-statement-admin-alter-ddl.md).
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 3]
```diff
@@ -91,6 +89,15 @@ The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the curre
...
     - `done`: indicates that the operation has been successfully executed on the TiDB owner node, but other TiDB nodes have not yet synchronized the changes performed by this DDL job.
+- `COMMENTS`: contains other information for auxiliary diagnostic purposes.
+    - `ingest`: ingest task for accelerated adding index backfill configured via [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)。
+    - `txn`: basic transactional backfill.
+    - `txn-merge`: transactional backfilling with a temporary index that gets merged with the original index when the backfilling is finished.
+    - `DXF`: tasks executed with distributed frameworks configured via [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
+    - `service_scope`: the service scope of the TiDB node configured via [`tidb_service_scope`](/system-variables.md#tidb_service_scope-new-in-v740).
+    - `thread`: concurrency of backfill tasks. The initial value can be set with `tidb_ddl_reorg_worker_cnt`. It supports dynamic modification via `ADMIN ALTER DDL JOBS`.
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:98]:
```suggestion
    - `thread`: concurrency of backfill tasks. The initial value can be set with `tidb_ddl_reorg_worker_cnt`. It supports dynamic modification via [`ADMIN ALTER DDL JOBS`](sql-statements/sql-statement-admin-alter-ddl.md).
```

File: sql-statements/sql-statement-admin-alter-ddl.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
++--------+---------+------------+-----------+--------------+-----------+----------+-----------+----------------------------+----------------------------+----------------------------+--------+-----------------------+
+|    124 | test    | t          | add index | public       |         2 |      122 |         3 | 2024-11-15 11:17:06.213000 | 2024-11-15 11:17:06.213000 | 2024-11-15 11:17:08.363000 | synced | ingest, DXF, thread=8 |
++--------+---------+------------+-----------+--------------+-----------+----------+-----------+----------------------------+----------------------------+----------------------------+--------+-----------------------+
+1 row in set (0.01 sec)
+```
+
+## Synopsis
+
+```ebnf+diagram
+AdminPauseDDLStmt ::=
```
Comment (tangenta) [inline sql-statements/sql-statement-admin-alter-ddl.md:64]:
```suggestion
AdminAlterDDLStmt ::=
```

File: TOC.md [hunk 5]
```diff
@@ -730,6 +730,7 @@
     - SQL Statements
       - [Overview](/sql-statements/sql-statement-overview.md)
       - [`ADMIN`](/sql-statements/sql-statement-admin.md)
+      - [`ADMIN ALTER DDL JOBS`](sql-statements/sql-statement-admin-alter-ddl.md)
```
Comment (hfxsd) [inline TOC.md:733]:
```suggestion
      - [`ADMIN ALTER DDL JOBS`](/sql-statements/sql-statement-admin-alter-ddl.md)
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 6]
```diff
@@ -91,6 +89,15 @@ The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the curre
...
     - `done`: indicates that the operation has been successfully executed on the TiDB owner node, but other TiDB nodes have not yet synchronized the changes performed by this DDL job.
+- `COMMENTS`: contains other information for auxiliary diagnostic purposes.
+    - `ingest`: ingest task for accelerated adding index backfill configured via [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)。
+    - `txn`: basic transactional backfill.
+    - `txn-merge`: transactional backfilling with a temporary index that gets merged with the original index when the backfilling is finished.
+    - `DXF`: tasks executed with distributed frameworks configured via [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
+    - `service_scope`: the service scope of the TiDB node configured via [`tidb_service_scope`](/system-variables.md#tidb_service_scope-new-in-v740).
+    - `thread`: concurrency of backfill tasks. The initial value can be set with `tidb_ddl_reorg_worker_cnt`. It supports dynamic modification via [`ADMIN ALTER DDL JOBS`](sql-statements/sql-statement-admin-alter-ddl.md).
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:98]:
```suggestion
    - `thread`: concurrency of backfill tasks. The initial value can be set with `tidb_ddl_reorg_worker_cnt`. It supports dynamic modification via [`ADMIN ALTER DDL JOBS`](/sql-statements/sql-statement-admin-alter-ddl.md).
```

File: sql-statements/sql-statement-admin-alter-ddl.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+The following are examples of this statement:
+
+```sql
+ADMIN ALTER DDL JOBS 101 THREAD = 8;
+ADMIN ALTER DDL JOBS 101 BATCH_SIZE = 256;
+ADMIN ALTER DDL JOBS 101 MAX_WRITE_SPEED = '200MiB';
+ADMIN ALTER DDL JOBS 101 THREAD = 8, BATCH_SIZE = 256;
+```
+
+To view the current parameter values for a particular DDL job, you can execute `ADMIN SHOW DDL JOBS` and the results are displayed in the `COMMENTS` column:
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-alter-ddl.md:49]:
```suggestion
To view the current parameter values for a specific DDL job, you can execute `ADMIN SHOW DDL JOBS` and the results are displayed in the `COMMENTS` column:
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 8]
```diff
@@ -91,6 +89,15 @@ The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the curre
     - `pausing`: indicates that the operation is being paused.
     - `paused`: indicates that the operation has been paused. This state only appears when you use the [`ADMIN PAUSED DDL JOBS`](/sql-statements/sql-statement-admin-pause-ddl.md) command to pause the DDL job. You can use the [`ADMIN RESUME DDL JOBS`](/sql-statements/sql-statement-admin-resume-ddl.md) command to resume the DDL job.
     - `done`: indicates that the operation has been successfully executed on the TiDB owner node, but other TiDB nodes have not yet synchronized the changes performed by this DDL job.
+- `COMMENTS`: contains other information for auxiliary diagnostic purposes.
+    - `ingest`: ingest task for accelerated adding index backfill configured via [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)。
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:93]:
```suggestion
    - `ingest`: ingest tasks for accelerated adding index backfill configured via [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)。
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 9]
```diff
@@ -91,6 +89,15 @@ The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the curre
...
+    - `txn`: basic transactional backfill.
+    - `txn-merge`: transactional backfilling with a temporary index that gets merged with the original index when the backfilling is finished.
+    - `DXF`: tasks executed with distributed frameworks configured via [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:96]:
```suggestion
    - `DXF`: tasks executed with Distributed eXecution Framework (DXF) configured via [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
```

File: sql-statements/sql-statement-admin-alter-ddl.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+The following DDL job types support `ADMIN ALTER DDL JOBS`: `ADD INDEX`, `MODIFY COLUMN`, and `REORGANIZE PARTITION`. For other DDL job types, executing `ADMIN ALTER DDL JOBS` reports an error `unsupported DDL operation`.
+
+Currently, you can only modiy the parameters of a single DDL job by executing `ADMIN ALTER DDL JOBS`. Modifying the parameters of multiple IDs at the same time is not supported.
+
+The following are the supported parameters for different DDL jobs and their corresponding system variables:
+
+- `ADD INDEX`:
+    - `THREAD`: the concurrency of the DDL job. The initial value is set by `tidb_ddl_reorg_worker_cnt`.
+    - `BATCH_SIZE`: the batch size. The initial value is set by [`tidb_ddl_reorg_batch_size`](/system-variables.md#tidb_ddl_reorg_batch_size).
+    - `MAX_WRITE_SPEED`: the maximum bandwidth limit for importing index records into each TiKV. The initial value is set by [`tidb_ddl_reorg_max_write_speed`](/system-variables.md#tidb_ddl_reorg_max_write_speed-new-in-v850).
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-alter-ddl.md:26]:
```suggestion
    - `MAX_WRITE_SPEED`: the maximum bandwidth limit for importing index records into each TiKV. The initial value is set by [`tidb_ddl_reorg_max_write_speed`](/system-variables.md#tidb_ddl_reorg_max_write_speed-new-in-v850). Currently this parameter only works for jobs with `ADD INDEX` submitted and running after disabling [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-introduced-from-rev-v710-version).
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 11]
```diff
@@ -91,6 +89,15 @@ The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the curre
     - `pausing`: indicates that the operation is being paused.
     - `paused`: indicates that the operation has been paused. This state only appears when you use the [`ADMIN PAUSED DDL JOBS`](/sql-statements/sql-statement-admin-pause-ddl.md) command to pause the DDL job. You can use the [`ADMIN RESUME DDL JOBS`](/sql-statements/sql-statement-admin-resume-ddl.md) command to resume the DDL job.
     - `done`: indicates that the operation has been successfully executed on the TiDB owner node, but other TiDB nodes have not yet synchronized the changes performed by this DDL job.
+- `COMMENTS`: contains other information for auxiliary diagnostic purposes.
+    - `ingest`: ingest tasks for accelerated adding index backfill configured via [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)。
+    - `txn`: basic transactional backfill.
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:94]:
```suggestion
    - `txn`: transaction-based index backfill after [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) is disabled.
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 12]
```diff
@@ -91,6 +89,15 @@ The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the curre
...
+    - `txn-merge`: transactional backfilling with a temporary index that gets merged with the original index when the backfilling is finished.
+    - `DXF`: tasks executed with Distributed eXecution Framework (DXF) configured via [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
+    - `service_scope`: the service scope of the TiDB node configured via [`tidb_service_scope`](/system-variables.md#tidb_service_scope-new-in-v740).
+    - `thread`: concurrency of backfill tasks. The initial value can be set with `tidb_ddl_reorg_worker_cnt`. It supports dynamic modification via [`ADMIN ALTER DDL JOBS`](/sql-statements/sql-statement-admin-alter-ddl.md).
+    - `batch_size`: batch size of backfill tasks. The initial value can be set with `tidb_ddl_reorg_batch_size`. It supports dynamic modification via `ADMIN ALTER DDL JOBS`.
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:99]:
```suggestion
    - `batch_size`: batch size of backfill tasks. You can set the initial value by `tidb_ddl_reorg_batch_size`. It supports dynamic modification via `ADMIN ALTER DDL JOBS`.
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 13]
```diff
@@ -91,6 +89,15 @@ The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the curre
...
     - `done`: indicates that the operation has been successfully executed on the TiDB owner node, but other TiDB nodes have not yet synchronized the changes performed by this DDL job.
+- `COMMENTS`: contains other information for auxiliary diagnostic purposes.
+    - `ingest`: ingest tasks for accelerated adding index backfill configured via [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)。
+    - `txn`: basic transactional backfill.
+    - `txn-merge`: transactional backfilling with a temporary index that gets merged with the original index when the backfilling is finished.
+    - `DXF`: tasks executed with Distributed eXecution Framework (DXF) configured via [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
+    - `service_scope`: the service scope of the TiDB node configured via [`tidb_service_scope`](/system-variables.md#tidb_service_scope-new-in-v740).
+    - `thread`: concurrency of backfill tasks. The initial value can be set with `tidb_ddl_reorg_worker_cnt`. It supports dynamic modification via [`ADMIN ALTER DDL JOBS`](/sql-statements/sql-statement-admin-alter-ddl.md).
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:98]:
```suggestion
    - `thread`: concurrency of backfill tasks. You can set the initial value by `tidb_ddl_reorg_worker_cnt`. It supports dynamic modification via [`ADMIN ALTER DDL JOBS`](/sql-statements/sql-statement-admin-alter-ddl.md).
```

File: sql-statements/sql-statement-admin-alter-ddl.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+
+The following DDL job types support `ADMIN ALTER DDL JOBS`: `ADD INDEX`, `MODIFY COLUMN`, and `REORGANIZE PARTITION`. For other DDL job types, executing `ADMIN ALTER DDL JOBS` reports an error `unsupported DDL operation`.
+
+Currently, you can only modiy the parameters of a single DDL job by executing `ADMIN ALTER DDL JOBS`. Modifying the parameters of multiple IDs at the same time is not supported.
+
+The following are the supported parameters for different DDL jobs and their corresponding system variables:
+
+- `ADD INDEX`:
+    - `THREAD`: the concurrency of the DDL job. The initial value is set by `tidb_ddl_reorg_worker_cnt`.
+    - `BATCH_SIZE`: the batch size. The initial value is set by [`tidb_ddl_reorg_batch_size`](/system-variables.md#tidb_ddl_reorg_batch_size).
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-alter-ddl.md:25]:
```suggestion
    - `BATCH_SIZE`: the batch size. You can set the initial value by [`tidb_ddl_reorg_batch_size`](/system-variables.md#tidb_ddl_reorg_batch_size).
```

File: sql-statements/sql-statement-admin-alter-ddl.md [hunk 15]
```diff
@@ -22,8 +22,8 @@ The following are the supported parameters for different DDL jobs and their corr
 
 - `ADD INDEX`:
     - `THREAD`: the concurrency of the DDL job. The initial value is set by `tidb_ddl_reorg_worker_cnt`.
-    - `BATCH_SIZE`: the batch size. The initial value is set by [`tidb_ddl_reorg_batch_size`](/system-variables.md#tidb_ddl_reorg_batch_size).
-    - `MAX_WRITE_SPEED`: the maximum bandwidth limit for importing index records into each TiKV. The initial value is set by [`tidb_ddl_reorg_max_write_speed`](/system-variables.md#tidb_ddl_reorg_max_write_speed-new-in-v850).
+    - `BATCH_SIZE`: the batch size. You can set the initial value by [`tidb_ddl_reorg_batch_size`](/system-variables.md#tidb_ddl_reorg_batch_size).
+    - `MAX_WRITE_SPEED`: the maximum bandwidth limit for importing index records into each TiKV. The initial value is set by [`tidb_ddl_reorg_max_write_speed`](/system-variables.md#tidb_ddl_reorg_max_write_speed-new-in-v850). Currently this parameter only works for jobs with `ADD INDEX` submitted and running after disabling [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
```
Comment (Frank945946) [inline sql-statements/sql-statement-admin-alter-ddl.md:26]:
```suggestion
    - `MAX_WRITE_SPEED`: the maximum bandwidth limit for importing index records into each TiKV. The initial value is set by [`tidb_ddl_reorg_max_write_speed`](/system-variables.md#tidb_ddl_reorg_max_write_speed-new-in-v850). Currently, the above parameters only work for jobs with `ADD INDEX` submitted and running after disabling [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
```

**Review by Frank945946 — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

File: sql-statements/sql-statement-admin-alter-ddl.md [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+The following DDL job types support `ADMIN ALTER DDL JOBS`: `ADD INDEX`, `MODIFY COLUMN`, and `REORGANIZE PARTITION`. For other DDL job types, executing `ADMIN ALTER DDL JOBS` reports an error `unsupported DDL operation`.
+
+Currently, you can only modiy the parameters of a single DDL job by executing `ADMIN ALTER DDL JOBS`. Modifying the parameters of multiple IDs at the same time is not supported.
+
+The following are the supported parameters for different DDL jobs and their corresponding system variables:
+
+- `ADD INDEX`:
+    - `THREAD`: the concurrency of the DDL job. The initial value is set by `tidb_ddl_reorg_worker_cnt`.
+    - `BATCH_SIZE`: the batch size. You can set the initial value by [`tidb_ddl_reorg_batch_size`](/system-variables.md#tidb_ddl_reorg_batch_size).
+    - `MAX_WRITE_SPEED`: the maximum bandwidth limit for importing index records into each TiKV. The initial value is set by [`tidb_ddl_reorg_max_write_speed`](/system-variables.md#tidb_ddl_reorg_max_write_speed-new-in-v850). Currently this parameter only works for jobs with `ADD INDEX` submitted and running after disabling [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-alter-ddl.md:26]:
```suggestion
    - `MAX_WRITE_SPEED`: the maximum bandwidth limit for importing index records into each TiKV. The initial value is set by [`tidb_ddl_reorg_max_write_speed`](/system-variables.md#tidb_ddl_reorg_max_write_speed-new-in-v850). 
  Currently this parameter only works for jobs with `ADD INDEX` submitted and running after disabling [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).  
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 17]
```diff
@@ -55,7 +55,7 @@ OWNER_ADDRESS: 0.0.0.0:4000
 
 ### `ADMIN SHOW DDL JOBS`
 
-The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the current DDL job queue, including running and queuing tasks, as well as the latest ten results in the completed DDL job queue. The returned result fields are described as follows:
+The `ADMIN SHOW DDL JOBS` statement is used to view all the results in the current DDL job queue, including running and queuing tasks, as well as the recent results in the completed DDL job queue. The returned result fields are described as follows:
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:58]:
```suggestion
The `ADMIN SHOW DDL JOBS` statement is used to view the 10 jobs in the current DDL job queue, including running and pending jobs (if any), and the last 10 jobs in the executed DDL job queue (if any). The returned result fields are described as follows:
```

File: sql-statements/sql-statement-admin-show-ddl.md [hunk 18]
```diff
@@ -126,31 +133,32 @@ ADMIN SHOW DDL JOBS;
...
++--------+---------+------------+---------------------------------+----------------------+-----------+----------+-----------+----------------------------+----------------------------+----------------------------+----------+-------------+
+14 rows in set (0.00 sec)
 ```
 
 From the output above:
 
-- Job 59 is currently in progress (`STATE` of `running`). The schema state is currently in `write reorganization`, but will switch to `public` once the task is completed to note that the change can be observed publicly by user sessions. The `end_time` column is also `NULL` indicating that the completion time for the job is currently not known.
+- Job 565 is currently in progress (`STATE` of `running`). The schema state is currently in `write reorganization`, but will switch to `public` once the task is completed to note that the change can be observed publicly by user sessions. The `end_time` column is also `NULL` indicating that the completion time for the job is currently not known.
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-show-ddl.md:159]:
```suggestion
- Job 565 is currently in progress (`STATE` of `running`). The schema state is currently in `write reorganization`, but will switch to `public` once the job is completed to note that the change can be observed publicly by user sessions. The `end_time` column is also `NULL` indicating that the completion time for the job is currently not known.
```

File: sql-statements/sql-statement-admin-alter-ddl.md [hunk 19]
```diff
@@ -0,0 +1,84 @@
...
+
+Currently, you can only modiy the parameters of a single DDL job by executing `ADMIN ALTER DDL JOBS`. Modifying the parameters of multiple IDs at the same time is not supported.
+
+The following are the supported parameters for different DDL jobs and their corresponding system variables:
+
+- `ADD INDEX`:
+    - `THREAD`: the concurrency of the DDL job. The initial value is set by `tidb_ddl_reorg_worker_cnt`.
+    - `BATCH_SIZE`: the batch size. You can set the initial value by [`tidb_ddl_reorg_batch_size`](/system-variables.md#tidb_ddl_reorg_batch_size).
+    - `MAX_WRITE_SPEED`: the maximum bandwidth limit for importing index records into each TiKV. The initial value is set by [`tidb_ddl_reorg_max_write_speed`](/system-variables.md#tidb_ddl_reorg_max_write_speed-new-in-v850). 
+  Currently this parameter only works for jobs with `ADD INDEX` submitted and running after disabling [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).
```
Comment (hfxsd) [inline sql-statements/sql-statement-admin-alter-ddl.md:27]:
```suggestion
  Currently these parameters only work for jobs with `ADD INDEX` submitted and running after disabling [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710).  
```

File: sql-statements/sql-statement-admin-alter-ddl.md [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+AlterJobOptionList ::=
+    AlterJobOption ( ',' AlterJobOption )*
+
+AlterJobOption ::=
+    identifier "=" SignedLiteral
+```
+
+## MySQL compatibility
+
+`ADMIN ALTER DDL JOBS` statement is a TiDB extension to MySQL syntax.
```
Comment (lilin90) [inline sql-statements/sql-statement-admin-alter-ddl.md:80]:
```suggestion
The `ADMIN ALTER DDL JOBS` statement is a TiDB extension to MySQL syntax.
```

File: sql-statements/sql-statement-admin-alter-ddl.md [hunk 21]
```diff
@@ -0,0 +1,87 @@
+---
+title: ADMIN ALTER DDL JOBS
+summary: An overview of the usage of `ADMIN ALTER DDL JOBS` for the TiDB database.
+---
+
+# ADMIN ALTER DDL JOBS
+
+`ADMIN ALTER DDL JOBS` can be used to change the parameter of a running DDL job. For example:
```
Comment (lilin90) [inline sql-statements/sql-statement-admin-alter-ddl.md:8]:
```suggestion
The `ADMIN ALTER DDL JOBS` statement allows you to modify the parameter of a single running DDL job. For example:
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**